website/docs/getting-started/tutorials/customer-support-tool/build-the-dashboard.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 1
+---
+
+
+# Build the Dashboard
+
+In this section, you'll learn to:
+* Connect to the tickets database.
+* Fetch the ticket details from the database.
+* Build a dashboard using the Chart and Stats Box widget.
+* Display data from queries on the dashboard.
+
+## Connect the database
+
+1. On the *Entity Explorer* to the left of the screen, click on the **Dashboard** page you created in the [Setting up the application](/getting-started/tutorials/customer-support-tool#setting-up-the-application) section.
+
+2. In the **Explorer** tab, click the **+** icon next to **Datasources** to add a new datasource.
+
+3. Select **PostgreSQL** under the **Databases** section. This opens the page where you can configure the fields to connect to a PostgreSQL database. 
+
+4. Click the pencil icon next to the default database name on the top left and rename the database to `supportTickets`.
```

Comment:
Some places we are using names of icon and at some places we are embedding the icon. Can we keep this consistent, either we show icons or write icon names.

Reply:
Hmm so the reason for this is sometimes we don't get the icon in keyboard like the exact one in the platform.

Reply:
Should we then just start using the names?

Reply:
We can keep this up for discussion and do a subsequent PR.

Reply:
yeah we need to finalise and use one method